### ISSUE TITLE
Issues running playground locally

### ISSUE THREAD

**Issue by andheiberg** [prismagraphql/graphql-playground#571]
#### This issue pertains to the following package(s):

- [x] GraphQL Playground - Electron App
- [ ] GraphQL Playground HTML
- [x] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

#### What OS and OS version are you experiencing the issue(s) on?

Mac OSX 10.12.6

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?

Commit 32c873a660463bc06f921906e1d0bcb48d1b05e0

#### What is the expected behavior?

As a user with an GQL schema for internal use only I would like to run playground and be able to share queries with my co-workers without storing any data on external services (graphqlbin.com).

It wasn't fully clear to me from the docs wether graphqlbin is just a hosted version of this repo, but I believe so. Assuming this is the case I clone the repo and as per the readme I run:

```
$ yarn
$ cd packages/graphql-playground-react
$ yarn start
```

Yarn start fails because it's missing dependencies... sure:

```
$ yarn install
$ yarn start
```

Hurray I now have a running server!

The readme suggests visiting http://localhost:3000/middleware.html?endpoint=https://api.graph.cool/simple/v1/swapi but that returns a 404 effectively it looks like `Cannot GET /middleware.html`

Right let's try http://localhost:3000/ looks promising, but there's an error in the JS console preventing the playground UI from loading.

```
(index):529 Uncaught ReferenceError: GraphQLPlayground is not defined
    at (index):529
```

#### What is the actual behavior?

For playground to function.

#### What steps may we take to reproduce the behavior?

Described above.

**Comment by Huvik** [comment]
Its because of this [commit](https://github.com/graphcool/graphql-playground/commit/1dec823888d2bb61381e569f15ebf9f3e74b97cd) you can uncomment it and it will work

**Comment by andheiberg** [comment]
Why would someone do that? I assume that's a blunder? Should I submit a PR to fix it?

**Comment by Huvik** [comment]
I am not sure how to graphcool wants to proceed, with building right now. I think tim overlooked it and didn't test it properly :(

**Comment by andheiberg** [comment]
That being said the answer to my original question of wether it's possible to run playground without sending data to Graphcools server the answer is currently no.

I started creating a PR to make it configurable, but then I realised actually storing the queries and creating sharing urls has probably been moved outside of this repo.
 
It used to be possible but has been removed hint: https://github.com/graphcool/graphql-playground/commit/5f5c539717ac45d4deb485d4ac85123b372230f6#diff-aea59863ff7ed8965e0962cfe2c276f5R106

It's handled by some API endpoint that's not configurable https://github.com/graphcool/graphql-playground/commit/5f5c539717ac45d4deb485d4ac85123b372230f6#diff-aea59863ff7ed8965e0962cfe2c276f5R84

I'm happy to finish my PR to make it all configurable. But I wouldn't do it unless the implementation of that API endpoint is public and part of this repo.
